features/installer/install_command.feature

```diff
@@ -0,0 +1,21 @@
+@cli
+Feature: Sylius Install Feature
+  In order to install Sylius via CLI
+  As an Administrator
+  I want to run an installation command
+
+  Scenario: Running install setup command
+    When I run a command "sylius:install:setup"
+    Then I should see 'Please enter a currency code or just hit ENTER to use "USD". For example "GBP".'
```

Comment:
How about `'Please enter a currency code (e.g. "GBP") or just hit ENTER to use "USD".'`?